arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2016-02-03",
+		"title": "iotHubClient",
+		"description": "API for using the IoT Hub Features"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get an IotHub",
+				"description": "Get an IotHub",
+				"operationId": "IotHubResource_GetIotHub",
```

Comment:
operationId should be in Noun_Verb format. 
`IotHubResource_Get` would be much better than repeating IotHub in the methodname.
`client.IotHubResource.Get()`